src/console.cpp

```diff
@@ -106,22 +106,18 @@ void IConsolePrint(TextColour colour_code, const std::string &string)
 
 	/* Create a copy of the string, strip if of colours and invalid
```

Comment:
```suggestion
	/* Create a copy of the string, strip it of colours and invalid
```

while in the area...